core/src/main/java/org/apache/carbondata/core/metadata/schema/table/CarbonTable.java

```diff
@@ -1309,4 +1313,22 @@ public String getParentTableName() {
     return allIndexes;
   }
 
+  /**
+   * This method returns the list of indx/merge index files for a segment in carbonTable.
+   */
+  public Set<String> getIndexFilesListForSegment(Segment segment) throws IOException {
```

Comment:
I think this method belongs in SegmentFileStore.

Reply:
done